config/operator/clusterwide/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
+  - ../base
+
+patches:
+  - ./operator_Deployment_patch.yaml
```

Comment:
please use `snake_case_for_filenames.yaml`

And don't forget to add a new line at the end of the file (your editor should do this for you if your configure it properly)